webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -73,6 +74,23 @@ public function indexAction(Request $request): Response
                 }
             }
             $execdata['execid']['cssclass'] = 'execid';
+            $type = $execdata['type']['value'];
+            switch ($type) {
+                case 'compare':
+                    $execdata['icon']['icon'] = 'code-compare';
+                    break;
+                case 'compile':
+                    $execdata['icon']['icon'] = 'down-left-and-up-right-to-center';
```

Comment:
I prefer `language` over this but keep it if you like it.

Reply:
Language is fine by me.